<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arinc665Qt::Media::FileWidget</class>
 <widget class="QWidget" name="Arinc665Qt::Media::FileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fileGroupBox">
     <property name="title">
      <string>File</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>nameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="mediumNumberGroupBox">
        <property name="title">
         <string>Medium Number</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="mediumNumber">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="checkValueTypeGroupBox">
        <property name="title">
         <string>Check Value Type</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="checkValueType"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="detailsStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="Arinc665Qt::Media::RegularFileWidget" name="regularFilePage"/>
     <widget class="Arinc665Qt::Media::LoadWidget" name="loadPage"/>
     <widget class="Arinc665Qt::Media::BatchWidget" name="batchPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Arinc665Qt::Media::RegularFileWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">arinc665_qt/media/RegularFileWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Arinc665Qt::Media::LoadWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">arinc665_qt/media/LoadWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Arinc665Qt::Media::BatchWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">arinc665_qt/media/BatchWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>mediumNumberGroupBox</tabstop>
  <tabstop>mediumNumber</tabstop>
  <tabstop>checkValueTypeGroupBox</tabstop>
  <tabstop>checkValueType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
